packaging/ubuntu-16.04/changelog

```diff
@@ -1,3 +1,108 @@
+    - Split refresh into essential and non-essential snaps to speed up
+      and minimise snapd restarts
+    - Ship snap-debug-info.sh script used for system diagnostics
+    - Add refresh-observe access to /v2/snaps/{name} REST API
+    - Improve error messages when attempting to run non-existent snap
+    - Switch to -u UID:GID for strace-static
+    - Support enabling snapd logging with snap set system
+      debug.snapd.{log,log-level}
+    - Fix potential unexpected auto-refresh of snap on managed schedule
+    - Fix potential segfault by guarding against kernel command-line
+      changes on classic system
+    - Fix proxy entries in /etc/environment with missing newline that
+      caused later manual entries to not be usable
+    - Fix offline remodelling by ignoring prerequisites that will
+      otherwise be downloaded from store
+    - Fix devmode seccomp deny regression that caused spamming the log
+      instead of actual denies
+    - Fix snap state lock leak during refresh unlink
+    - Fix not re-pinning validation sets that were already pinned when
+      enforcing new validation sets
+    - Fix handling of unexpected snapd runtime failure
+    - Fix /v2/notices REST API skipping notices with duplicate
+      timestamps
+    - Fix comparing systemd versions that may contain pre-release
+      suffixes
+    - Fix udev potentially starting before snap-device-helper is made
+      available
+    - Fix race in snap seed metadata loading
+    - Fix treating cloud-init exit status 2 as error
+    - Fix to prevent sending refresh complete notification if snap
+      refresh-observe is connected
+    - Fix to queue default-configure service commands run after the hook
+      to ensure it gets up-to-date config values
+    - Fix stop service failure when the service is not actually running
+      anymore
+    - Add registry interface that provides snaps access to a particular
+      registry view
+    - home interface: autoconnect home on Ubuntu Core Desktop
+    - serial-port interface: support RPMsg tty
+    - display-control interface: allow changing LVDS backlight power and
+      brightness
+    - power-control interface: support for battery charging thesholds,
+      type/status and AC type/status
+    - cpu-control interface: allow CPU C-state control
+    - raw-usb interface: support RPi5 and Thinkpad x13s
+    - custom-device interface: allow device file locking
+    - lxd-support interface: allow LXD to self-manage its own cgroup
+    - network-manager interface: support MPTCP sockets
+    - network-control interface: allow plug/slot access to gnutls config
+      and systemd resolved cache flushing via D-Bus
+
+ -- Ernest Lotter <ernest.lotter@canonical.com>  Tue, 16 Jul 2024 16:39:06 +0200
+
+snapd (2.64) xenial; urgency=medium
+
+  * New upstream release, LP: #2072986
+    - TODO: Specified content
+
+ -- Ernest Lotter <ernest.lotter@canonical.com>  Mon, 15 Jul 2024 10:47:31 +0200
+
```

Comment:
Here too?

Reply:
thanks, this is due to generating twice, once we are happy with content, will generate from scratch to prevent duplicates...

Reply:
fixed